v2/apiserver/config.go

```diff
@@ -22,9 +22,9 @@ import (
 	"golang.org/x/oauth2"
 )
 
-// database returns a *mongo.Database connection based on configuration obtained
-// from environment variables.
-func database(ctx context.Context) (*mongo.Database, error) {
+// databaseConfig returns a *mongo.Database connection based on configuration
+// obtained from environment variables.
+func databaseConfig(ctx context.Context) (*mongo.Database, error) {
```

Comment:
I see this change was made for consistency with the other function names probably, but it might be misleading as it doesn't return configuration, rather it returns a database connection.

Reply:
That's reasonable.  How about `databaseConnection`?

Reply:
sgtm